spartan/aztec-network/values/ci-sepolia.yaml

```diff
@@ -4,9 +4,6 @@ aztec:
   realProofs: false
   l1DeploymentMnemonic: ""
   numberOfDefaultAccounts: 0
-  validatorKeyIndexStart: 0
-  proverKeyIndexStart: 3
-  botKeyIndexStart: 4
```

Comment:
I understand the idea is to just rely on the default values file, right?

Reply:
yep 👍